core/lib/globals_api.h

```diff
@@ -854,6 +854,13 @@ enum {
      * their application values, as required for DR's internal block mangling.
      */
     DR_NOTE_REG_BARRIER,
+    /* Used for translation from an instruction list.  These apply not only to
+     * client-inserted clean calls but all inserted calls whether inserted by
+     * clients or DR and whether fully clean or not.  This is thus distinct from
+     * DR_NOTE_CLEAN_CALL_END.
+     */
+    DR_NOTE_CALLOUT_SEQUENCE_START,
+    DR_NOTE_CALLOUT_SEQUENCE_END,
```

Comment:
It looks like DR_NOTE_FIRST_RESERVED only gives DR 16 reserved note values?  I'm hoping the instr_t.offset change will open up the use of labels for translation support for all the clean all varieties and subsections: but if those all use top-level labels we'll run out.  Should have a single call label and put sub-labels inside the label data area?  Should we change the reservation value to give us a few hundred at least if not thousands -- I really doubt that's much of a compat break, and if we're already bumping the version we could do that now.

Reply:
The only concern I can think of is if some client was reserving a very large range. But I agree it should be fine to extend  DR_NOTE_FIRST_RESERVED to give core DR more. If there is some conflict then `drmgr_reserve_note_range` would inform the user.

Reply:
Done: reduced the value to give DR thousands.  Added a release note about this.